WBooks/Library/LibraryController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  LibraryController.swift
+//  WBooks
+//
+//  Created by Nacho Mendez on 04/08/2021.
+//
+
+import UIKit
+
+class LibraryController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private lazy var library: LibraryView = LibraryView()
+    
+    override func loadView() {
+        view = library
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // NavigationBar
+        configurationNavigationBar()
+        
+        // delegares and data source
+        library.tableBooks.delegate = self
+        library.tableBooks.dataSource = self
+        
+        // cell .xib
+        let nib = UINib.init(nibName: "MyCustomCell", bundle: nil)
+        library.tableBooks.register(nib, forCellReuseIdentifier: "MyCustomCell")
+    }
+    
+    // func for the NavigationBar config
+    func configurationNavigationBar() {
+        
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: UIImage(named: "Navigation baric_search"), style: .done, target: self, action: nil)
+        
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "Navigation baric_notifications"), style: .done, target: self, action: nil)
+        
+        let imageBackground = UIImage(named: "bc_nav bar")
+        navigationController?.navigationBar.setBackgroundImage(imageBackground, for: .default)
+        
+        self.navigationItem.rightBarButtonItem?.tintColor = UIColor.white
+        
+        self.navigationItem.leftBarButtonItem?.tintColor = UIColor.white
+        
+        //navigationController?.title = "LIBRARY"
+        
+        //title = "LIBRARY"
+        title = NSLocalizedString("TITLE_NAVBAR", comment: "")
```

Comment:
Let's add a `comment:` there. It's just some info about what the string represents. For instance: `"Title at the top of the Library view"`